File: Pension-Management-Backend/Process-Pension-Microservice/src/main/java/com/cognizant/process/pension/model/PensionDetail.java
package com.cognizant.process.pension.model;

import java.sql.Date;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PensionDetail {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	private String name;
	private Date dateOfBirth;
	private String panNumber;
	private String pensiontype;
	private Double pensionAmount;
	
	public PensionDetail(String name, Date dateOfBirth, String panNumber, String pensiontype, Double pensionAmount) {
		super();
		this.name = name;
		this.dateOfBirth = dateOfBirth;
		this.panNumber = panNumber;
		this.pensiontype = pensiontype;
		this.pensionAmount = pensionAmount;
	}
}

File: Pension-Management-Frontend/Pension-Management-Portal/src/app/model/pension-detail.ts
export class PensionDetail {
    public id: number;
	public name: string;
	public dateOfBirth: Date;
	public panNumber: string;
	public pensiontype: string;
	public pensionAmount: number;

    constructor() {
        this.id = 0;
        this.name = '';
        this.dateOfBirth = new Date();
        this.panNumber = '';
        this.pensiontype = '';
        this.pensionAmount = 0;
    }
}